src/main/java/profiler/profiling/InputFeedProfiling.java

```diff
@@ -0,0 +1,96 @@
+package profiler.profiling;
+
+import algebra.curves.barreto_naehrig.bn254a.BN254aFields.BN254aFr;
+import algebra.curves.barreto_naehrig.bn254a.BN254aG1;
+import algebra.curves.barreto_naehrig.bn254a.BN254aG2;
+import algebra.curves.barreto_naehrig.bn254a.BN254aGT;
+import algebra.curves.barreto_naehrig.bn254a.BN254aPairing;
+import algebra.curves.barreto_naehrig.bn254a.bn254a_parameters.BN254aFrParameters;
+import algebra.curves.barreto_naehrig.bn254a.bn254a_parameters.BN254aG1Parameters;
+import algebra.curves.barreto_naehrig.bn254a.bn254a_parameters.BN254aG2Parameters;
+import algebra.fields.Fp;
+import configuration.Configuration;
+import input_feed.distributed.TextToDistributedR1CS;
+import org.apache.spark.api.java.JavaPairRDD;
+import relations.objects.Assignment;
+import relations.r1cs.R1CSRelationRDD;
+import scala.Tuple2;
+import zk_proof_systems.zkSNARK.DistributedProver;
+import zk_proof_systems.zkSNARK.DistributedSetup;
+import zk_proof_systems.zkSNARK.Verifier;
+import zk_proof_systems.zkSNARK.objects.CRS;
+import zk_proof_systems.zkSNARK.objects.Proof;
+
+public class InputFeedProfiling {
+//    private TextToDistributedR1CS<BN254aFr> converter;
```

Comment:
🚧 

Maybe remove all the commented lines?

Reply:
This thing isn't even close to ready yet.